frog/imports/ui/GraphEditor/utils/index.js

```diff
@@ -19,43 +19,71 @@ export const between = (
   return Math.min(Math.max(x, minval), maxval);
 };
 
-export class TextInput extends Component {
-  constructor(props: { value: string }) {
-    super(props);
-    this.state = { val: this.props.value || '' };
-  }
-
-  state: { val: string };
-
-  componentDidMount() {
-    this.textInput.focus();
-  }
-
-  onChange = (e: any) => this.setState({ val: e.target.value });
-
-  onSubmit = (e: any) => {
-    e.preventDefault();
-    this.props.onSubmit(this.state.val);
-  };
-  textInput: { focus: Function };
-
-  handleKey = (e: any) => {
-    if (e.keyCode === 27) {
-      this.props.onCancel();
+export const S = (obj, options) => {
```

Comment:
Why do you need this function here? Is it for debugging purposes or is it used somewhere in the code?